components/component.css

```diff
@@ -11,6 +11,7 @@
 @import url(/components/grid.css);
 @import url(/components/modal.css);
 @import url(/components/rating.css);
+@import url(/components/navbar.css);
 
```

Comment:
Remove extra spaces.

Reply:
Hey Purva, the space is for differentiating components and other classes.

Reply:
Ok, I suggest adding a comment to differentiate.